index.html

```diff
@@ -6549,7 +6579,7 @@ <h5>Presentational Roles Conflict Resolution</h5>
 			<div class="role-description">
 				<p>A perceivable <rref>section</rref> containing content that is relevant to a specific, author-specified  purpose and sufficiently important that users will likely want to be able to navigate to the section easily and to have it listed in a summary of the page. Such a page summary could be generated dynamically by a user agent or assistive technology.</p>
 				<p>Authors SHOULD limit use of the region role to sections containing content with a purpose that is not accurately described by one of the other <rref>landmark</rref> roles, such as <rref>main</rref>, <rref>complementary</rref>, or <rref>navigation</rref>.</p>
-				<p>Authors MUST give each element with role region a brief label that describes the purpose of the content in the region. Authors SHOULD reference a visible label with <pref>aria-labelledby</pref> if a visible label is present. Authors SHOULD include the label inside of a heading whenever possible. The heading MAY be an instance of the standard host language heading element or an instance of an element with role <rref>heading</rref>.</p>
+				<p>Authors MUST give each element with role region a brief label that describes the purpose of the content in the region. For host languages that support elments with the default role of region (e.g. HTML <code>section</code>), an explict region role must be defined (e.g. <code>role="region"</code>) to enable the nameFrom: heading technique.</p>
```

Comment:
couple typo fixes and rewording suggestion to remove the need for two e.g.,'s in the sentence, and to swap out the instance of the lowercase 'must'.
```suggestion
				<p>Authors MUST give each element with a region role a brief label that describes the purpose of the region. For host languages that support elements with the default role of region (e.g., HTML <code>section</code>), authors would need to specify an explicit <code>role="region"</code> on the element to enable the nameFrom: heading technique.</p>
```

Reply:
A similar rule should be added to `role=complementary` per the HTML AAM change to `<aside>` where it should only map to `role=complementary` if it is scoped to the `<body>` or `<main>` element.